tests/filecheck/parser-printer/graph_region.mlir

```diff
@@ -53,6 +53,16 @@ builtin.module {
 
 // -----
 
+// A graph region that refers to values that are not defined in the module.
+
+builtin.module {
+    %0 = "test.termop"(%1, %2) : (i32, i32) -> i32
+}
+
+// CHECK: values %1, %2 were used but not defined
```

Comment:
Can the check statement go before the code? I'm also not sure the comment is necessary, the error message describes what is going on.

Reply:
moved it, thanks

Reply:
Unless github is being weird, it seems you moved a different one?

Reply:
I think they are similar except the first one only reports one value, so it's coming from a different if branch in core.py